public/js/sticky-sidebar.js

```diff
@@ -0,0 +1,38 @@
+/* global $ */
```

Comment:
This seems to break the code for some reason. I can't work out exactly why. As none of the other modules are written like this I fear it's going to add a lot more work than is needed for this so maybe we can open an issue to refactor all of the JS at some point instead?